reladomographql/src/test/resources/test-schema.graphqls

```diff
@@ -0,0 +1,541 @@
+type Mutation
+{
+    testBalanceNoAcmap_insert(testBalanceNoAcmap: TestBalanceNoAcmapInput): TestBalanceNoAcmap
+    order_insert(order: OrderInput): Order
+    product_insert(product: ProductInput): Product
+    orderItem_insert(orderItem: OrderItemInput): OrderItem
+}
+
+
+type Query
+{
+    testBalanceNoAcmapById( accountId: String!  productId: Int!  positionType: Int! ): TestBalanceNoAcmap
+    testBalanceNoAcmaps(findMany: TestBalanceNoAcmapFinder filter: TestBalanceNoAcmapFinder where: TestBalanceNoAcmapFinder order_by: TestBalanceNoAcmapOrderBy limit: Int): [TestBalanceNoAcmap]
```

Comment:
This answers some of my previous questions about how the NameById queries work. They are generated to required the entire primary key, so a single result really is expected.

Separately, what is the difference between `findMany` and `filter` and `where`?